### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by gauraveluminous** [add2cal/add-to-calendar-button#470]
### What happened?

Hello, I have tried to implement it on an angular local project and it seems to be working fine. But when I tried to upload code on the server, it was not showing. also not showing any kind of error in the console or during compilation as well. What should be the reason ?

My code :

HTML code : 

```
<span *ngIf="showCal">
                <add-to-calendar-button
                [attr.name]="eventTitle"
                options="'Apple', 'Google', 'iCal','Outlook.com', 'Yahoo'"
                [attr.location]="eventLocation"
                [attr.startDate]="calenderStartDate  "
                [attr.endDate]="calenderEndDate"
                [attr.startTime]="eventStartTime"
                [attr.endTime]="eventendtTime"
                [attr.description]="eventDescription"
                [attr.label]="'buttons.ADD_TO_CALENDAR' | translate"
              ></add-to-calendar-button>
</span>
```
TS code:

```
import 'add-to-calendar-button';

        this.eventTitle = this.eventDetails.title
        this.eventLocation = this.eventDetails.location
        this.calenderStartDate  = this.datePipe.transform(this.eventDetails.cal_start_date , 'yyyy-MM-dd')
        this.calenderEndDate  = this.datePipe.transform(this.eventDetails.cal_end_date , 'yyyy-MM-dd')
        this.eventStartTime  = this.datePipe.transform(this.eventDetails.start_date, 'hh:mm');
        this.eventendtTime  = this.datePipe.transform(this.eventDetails.end_date, 'hh:mm');
        this.eventDescription = this.eventDetails.description

  if (this.eventDetails.status == 0) {
          this.eventCancelFlag = true;
        }
        if (Date.parse(this.eventDetails.end_date) < this.today) {
          this.expiryFlag = true;
        }

        this.showCal = (this.eventCancelFlag || this.expiryFlag) ? false : true;
```

### How to reproduce?

on server its not showing add-to-calender-button

### System & Environment

1)Windows 
2)Chrome

**Comment by jekuer** [comment]
@gauraveluminous , what does it tell you, if you add the option "debug"?
There are 2 additional spaces at the startDate - could be an issue, maybe not.
Please also compare to the example at https://stackblitz.com/edit/add-to-calendar-button-angular?file=src%2Fapp%2Fapp.component.css,src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.module.ts,src%2Fmain.ts,src%2Fapp%2Fapp.component.html

**Comment by gauraveluminous** [comment]
Hello, It is working now on the server. Actually, I missed importing on particular component file. though space in the start date is not an issue. thanks for the prompt response too. @jekuer .  Closing the bug.  For others if they face such an issue just to mention that without =>  import 'add-to-calendar-button';  it was working locally too. since I have already added schema to a particular module, but in order to work on the server it is required.
